profiler/src/Demos/Samples.Computer01/SocketTimeout.cs

```diff
@@ -139,7 +139,7 @@ private void EnsureOperationTimesOut(Action<Socket> action, string operationName
             }
         }
 
-        private async Task StartServerAsync(object obj)
+        private void StartServerAsync(object obj)
```

Comment:
Is a method really async if you `GetAwaiter().GetResult()` 😛

Reply:
Good point. Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/5648/commits/4201d8e2929608e08034ebd64263b527f91d210c